fetch.bs

```diff
@@ -209,7 +209,7 @@ number.
 <h3 id=url>URL</h3>
 
 <p>A <dfn export>local scheme</dfn> is a <a for=url>scheme</a> that is "<code>about</code>",
-"<code>blob</code>", "<code>data</code>", or "<code>filesystem</code>".
+"<code>blob</code>" or "<code>data</code>".
```

Comment:
Please keep the Oxford comma before "or".

Reply:
will keep comma